src/ProductDefects.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\ShopStandards\UsedGoods.
```

Comment:
I think we need to decide if it is `UsedGoods` or `ProductDefects`, to me the first option sounds more descriptive, in any case it should be one name in the comments, class and file names.

Reply:
Although UsedGoods might be more generic in natural language, I would recommend to go with ProductDefects, because it contains more relevant keywords in the code space.

Reply:
In any case this was already defined in the ticket description. I forgot to change this one (it started with naming everything UsedGoods until I noticed).